proposals/open-telemetry/pom.xml

```diff
@@ -0,0 +1,17 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.eclipse.microprofile</groupId>
+    <artifactId>microprofile-parent</artifactId>
+    <version>2.1</version>
+  </parent>
+  <groupId>org.eclipse.microprofile.opentracing</groupId>
```

Comment:
```suggestion
  <groupId>org.eclipse.microprofile.opentelemetry</groupId>
```

Reply:
We were discussing this and we'll keep `org.eclipse.microprofile.opentracing` for the group id and opentelemetry-* for the artifacts.
In the future this will likely be a multi-module spec.